examples/DancingGoat/appsettings.json

```diff
@@ -14,7 +14,8 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "CMSConnectionString": "Data Source=GIGACHAD;Initial Catalog=xperience-lucene;Integrated Security=True;Persist Security Info=False;Connect Timeout=60;Encrypt=False;Current Language=English;"
+    "CMSConnectionString": ""
   },
-  "CMSHashStringSalt": "sal"
+  "CMSHashStringSalt": "sal",
+  "WebCrawlerBaseUrl": ""
```

Comment:
Is this setting necessary for successful setup of the example project? Do we have any documentation covering this setting?

Reply:
It is necessary for advanced examples. I will add documentation for this.